src/main/java/controller/planning/EventPlanningController.java

```diff
@@ -92,6 +98,9 @@ private void createUsersInDatabase(List<UserView> listOfUsers) {
                 e.printStackTrace();
             }
             users_id.add(tempUser.getId());
+            for(ThingView thing: user.getThings()){
+                mapOfThingsToWriteIntoDB.put(thing, tempUser.getId());
```

Comment:
Mixed responsibilities. It shouldn't be here. Consider saving list of things in different method, `saveUsersThings` for example